R/pkg/R/DataFrame.R

```diff
@@ -2121,7 +2121,7 @@ setMethod("arrange",
           })
 
 #' @rdname arrange
-#' @name orderBy
```

Comment:
@name is actually an required field for roxygen - so it will just going to infer that from the function signature right after this comment block, if it is not specified.
I'm surprised about the duplicate alias though - do you mean @aliases or something else?